packages/plugin-ext/src/api/plugin-api.ts

```diff
@@ -163,6 +164,8 @@ export interface PluginManagerExt {
     $init(pluginInit: PluginInitData, configStorage: ConfigStorage): PromiseLike<void>;
 
     $updateStoragePath(path: string | undefined): PromiseLike<void>;
+
+    $activateByEvent(event: string): Promise<void>;
```

Comment:
[minor]: Why isn't it a `PromiseLike` like every other return type on this interface?

Reply:
`PromiseLike` was introduced in `theia.d.ts`, and i guess by copy paste it got in plugin-api and other files as well. I don't know why `PromiseLike` should be used here, except historical reasons.

Reply:
OK, I am fine with it. Thanks!